cmd/release-controller-api/http.go

```diff
@@ -1098,6 +1103,7 @@ func (c *Controller) httpReleases(w http.ResponseWriter, req *http.Request) {
 	).Parse(releasePageHtml))
 
 	imageStreams, err := c.releaseLister.List(labels.Everything())
+
```

Comment:
Just a nit, but i tend to keep err checks nestled with the code that it is checking.

Reply:
don't forget this one

Reply:
Fixed.